Actions/AL-Go-Helper.ps1

```diff
@@ -455,6 +455,11 @@ function ReadSettings {
         "keyVaultClientIdSecretName"             = ""
         "codeSignCertificateUrlSecretName"       = "codeSignCertificateUrl"
         "codeSignCertificatePasswordSecretName"  = "codeSignCertificatePassword"
+        "codeSignKeyvaultUriSecretName"          = ""
```

Comment:
I think we should have one secret called azureCodeSignContext, which should be a compressed json structure with these fields (without the codeSign part)

Reply:
I was trying to avoid doing that as GitHub specifically discourages the use of [structured data in secrets](https://docs.github.com/en/actions/security-guides/encrypted-secrets). What is the downside of having the values as separate secrets?

Reply:
I know, and the primary reason for that is that only the entire secret is safe - meaning that if you add a space to the structure or like, the secret might be exposed.
But... - we actually add every property to the masked value and is not hurt by that.
Reason for the structure is that everything else in AL-Go is using this and it is much easier to maintain one secret than 6 - even though the one secret is a structure.
Let's discuss pros and cons on Wednesday if needed.